src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AbstractRazorDelegatingEndpoint.cs

```diff
@@ -34,10 +34,7 @@ internal abstract class AbstractRazorDelegatingEndpoint<TRequest, TResponse> : I
             Logger = logger ?? throw new ArgumentNullException(nameof(logger));
         }
 
-        /// <summary>
-        /// The delegated object to send to the <see cref="CustomMessageTarget"/>
-        /// </summary>
-        protected abstract IDelegatedParams? CreateDelegatedParams(TRequest request, RazorRequestContext razorRequestContext, Projection projection, CancellationToken cancellationToken);
+        protected virtual bool OnlySingleServer { get; } = true;
```

Comment:
can this just be a parameter to the constructor?

Reply:
I didn't want to add the burden to every inheritor, since I expect it will only be used once.

Reply:
Oh I meant as an optional param with default of true. Just personal taste I think though

Reply:
Oh yeah, that would work. Personally I prefer this, as it stands out more in the inherited class. If the constructor was smaller I could maybe see it working, though I'm generally not much of a fan of optional parameters on non-public code.